odoo/tools/misc.py

```diff
@@ -452,27 +452,22 @@ def get_iso_codes(lang: str) -> str:
 
 
 def scan_languages() -> list[tuple[str, str]]:
-    """ Returns all languages supported by OpenERP for translation
+    """ Returns all languages supported for translation
```

Comment:
You could move this function from tools to web. It is used only there just like `list_countries`.